dashboard/src/components/AppView/AccessURLTable/AccessURLItem/AccessURLItem.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+
+import { IResource, IServiceSpec, IServiceStatus } from "../../../../shared/types";
+import "./AccessURLItem.css";
+
+interface IAccessURLItem {
+  service: IResource;
+}
+
+class AccessURLItem extends React.Component<IAccessURLItem> {
+  // isLink returns true if there are any link in the Item
+  get isLink(): boolean {
```

Comment:
>  the Service IResource should

I do not agree with that statement, isLink is very specific to this component rendering logic so I can not see how it would fit in the model.

W.R.T the getters, I was the one that suggested it to @andresmgot and we just had a chat about it. I personally have not put much thought on it but he convinced me that actually using a function (as you also suggested) is better so it does not get mistaken by a static property.

About making them private functions, that means that we will not (and should not) test them, I guess that's ok.

Reply:
> I do not agree with that statement, isLink is very specific to this component rendering logic so I can not see how it would fit in the model.

Why not? Shouldn't it be possible to generally tell if an IService has a link? It is a function that acts directly on the IService spec to figure it out, the component has to use the prop to get that value. In any case, I think it's arguable but I agree and I think my bigger concern here was not using getters/setters because they look too much like static properties.

Good point about private functions.